### ISSUE TITLE
Display VCS Revision for dev-xyz

### ISSUE THREAD

**Issue by TekkCraft** [pyrech/composer-changelogs#62]
Currently, when updating an unstable version, it only shows ``dev-develop to dev-develop`` for example, as in the following snippet:

````
 - squizlabs/php_codesniffer updated from 3.4.2 to 3.5.0
   See changes: https://github.com/squizlabs/PHP_CodeSniffer/compare/3.4.2...3.5.0
   Release notes: https://github.com/squizlabs/PHP_CodeSniffer/releases/tag/3.5.0

 - friendsofcake/bootstrap-ui updated from dev-develop to dev-develop
   See changes: https://github.com/FriendsOfCake/bootstrap-ui/compare/958a5dd...6d57476
````

But it would carry more information  if it would also display the VCS revision in that line, like so:

````
 - squizlabs/php_codesniffer updated from 3.4.2 to 3.5.0
   See changes: https://github.com/squizlabs/PHP_CodeSniffer/compare/3.4.2...3.5.0
   Release notes: https://github.com/squizlabs/PHP_CodeSniffer/releases/tag/3.5.0

 - friendsofcake/bootstrap-ui updated from dev-develop~958a5dd to dev-develop~6d57476
   See changes: https://github.com/FriendsOfCake/bootstrap-ui/compare/958a5dd...6d57476
````

Alternatively to the tilde sign, it could use a slash, instead:

````
 - friendsofcake/bootstrap-ui updated from dev-develop/958a5dd to dev-develop/6d57476
````

Or the at sign (preferred):

````
 - friendsofcake/bootstrap-ui updated from dev-develop@958a5dd to dev-develop@6d57476
````

**Comment by pyrech** [comment]
Hi,

Thanks for your suggestion :slightly_smiling_face: Like you, I prefer the use of the `@` sign. I think this modification should not be to complicated to do (probably just a matter of updating the class [Version](https://github.com/pyrech/composer-changelogs/blob/master/src/Version.php)).

Would you mind to give it a try?

**Comment by laurent35240** [comment]
Hi,

I created this PR #64 for fixing this issue.

**Comment by pyrech** [comment]
Fixed in #64
